src/subscription_manager/repofile.py

```diff
@@ -469,6 +474,14 @@ def fix_content(self, content):
             apt_cont['Suites'] = 'default'
             apt_cont['Components'] = 'all'
             apt_cont['Trusted'] = 'yes'
+
+            if apt_cont['arches'] == None:
+                apt_cont['arches'] = 'none'
+            else:
+                arches_str = ", ".join(apt_cont['arches']).lower()
```

Comment:
is the `lower()` actually needed? my debian knowledge tells me all the architecture names (even for non-Linux archs) are lowercase already; even if they were, we ought to use them as they are

Reply:
I thought it would be more consistent to make it all lowercase. Nevertheless, fixed as suggested.